homeassistant_cli/const.py

```diff
@@ -19,3 +19,4 @@
     ('CHANGED', 'last_changed'),
 ]
 COLUMNS_SERVICES = [('DOMAIN', 'domain'), ("SERVICE", "domain.services[*]")]
+COLUMNS_WIDTH_STR = "..."
```

Comment:
Was considering to replace `...` with the horizontal ellipsis `\u2026`. Thoughts?

Reply:
should be okey I think.